Makefile

```diff
@@ -32,7 +32,7 @@ export INSTALL_DEFAULT_CATALOGS := true
 # By default setup-envtest will write to $XDG_DATA_HOME, or $HOME/.local/share if that is not defined.
 # If $HOME is not set, we need to specify a binary directory to prevent an error in setup-envtest.
 # Useful for some CI/CD environments that set neither $XDG_DATA_HOME nor $HOME.
-SETUP_ENVTEST_BIN_DIR_OVERRIDE=
+SETUP_ENVTEST_BIN_DIR_OVERRIDE := --bin-dir $(ROOT_DIR)/bin/envtest-binaries
 ifeq ($(shell [[ $$HOME == "" || $$HOME == "/" ]] && [[ $$XDG_DATA_HOME == "" ]] && echo true ), true)
 	SETUP_ENVTEST_BIN_DIR_OVERRIDE += --bin-dir /tmp/envtest-binaries
```

Comment:
could the += cause problems here and we end up with SETUP_ENVTEST_BIN_DIR_OVERRIDE = --bin-dir /$(ROOT_DIR)/... --bin-dir /tmp/envtest-binaries 

Is this what we are after?

Reply:
I do not think so, but we can replace `:=` with `+=`
It will not change the result 👍

Reply:
Done